.github/workflows/release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Release
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set env
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: otel/opentelemetry-go-agent:${{ steps.vars.outputs.tag }}
```

Comment:
I suggest avoiding the "Agent" name as it brings confusion. More https://github.com/open-telemetry/opentelemetry.io/issues/1689

can we use `auto` or maybe even `auto-instrumentation` instead?

Reply:
If the community like including `auto`, we could mimic the operator's style and use `otel/autoinstrumentation-go`.

Reply:
Updated name to `autoinstrumentation-go`.